Lib/dis.py

```diff
@@ -609,6 +610,8 @@ def get_argval_argrepr(self, op, arg, offset):
                     argrepr = obj.__name__
                 else:
                     argrepr = repr(obj)
+            elif deop == LOAD_SPECIAL:
+                argrepr = get_special_method_names()[arg]
```

Comment:
This creates the list every time. With the intrinsic descs we cache it in opcode.py.

Reply:
Good point, fixed